web/src/settings_users.js

```diff
@@ -199,6 +224,22 @@ function human_info(person) {
     return info;
 }
 
+function create_role_filter_dropdown($events_container) {
+    new dropdown_widget.DropdownWidget({
+        widget_name: "select_role",
```

Comment:
We should use a more specific name here, like `user_list_select_user_role`.

Reply:
Oh! We need to pass different widget_name for dropdown in users list and deactivated users list, otherwise it will create two widgets with the same id.

Reply:
Fixed both